### PR TITLE
fix: Added updates to files

### PR DESCRIPTION
**Pull request by hyperxtend**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,15 +34,15 @@
     ]
   },
   "devDependencies": {
+    "@testing-library/jest-dom": "^5.5.0",
+    "bootstrap": "^4.4.1",
```
Comment (strange-developer) [inline package.json]: Should this be here?

File: src/components/board/board.test.jsx [hunk 2]
```diff
@@ -1,19 +1,21 @@
 import React from 'react';
-import Board from './board';
 import { shallow, mount } from 'enzyme';
+import Board from './board';
 
-it('<Board/> renders without error', () => {
-  let squares = Array(9).fill(null);
-  shallow(<Board squares={squares} />);
-});
+describe('Board', () => {
```
Comment (strange-developer) [inline src/components/board/board.test.jsx]: A small convention that we follow at GK in our describe for components is using the following syntax:
```js
describe('<Board />', () => {
```

File: src/components/board/board.test.jsx [hunk 3]
```diff
@@ -1,19 +1,21 @@
...
+  it('renders without error', () => {
+    const squares = Array(9).fill(null);
+    shallow(<Board squares={squares} />);
```
Comment (strange-developer) [inline src/components/board/board.test.jsx]: A better test here would be to add an assertion to make sure that `<Board />` exists.

File: src/components/game/game.test.jsx [hunk 4]
```diff
@@ -1,46 +1,52 @@
...
-  const wrapper = mount(<Game />);
-  const firstPlayer = wrapper
-    .find('div.gameInfo')
-    .children()
-    .first()
-    .text();
-  expect(firstPlayer).toEqual('Next player is X');
+describe('Game', () => {
+  it('renders without error', () => {
+    shallow(<Game />);
```
Comment (strange-developer) [inline src/components/game/game.test.jsx]: A better test here would be checking that `<Game />`  exists

File: src/components/game/game.test.jsx [hunk 5]
```diff
@@ -1,46 +1,52 @@
...
 
-it('<Game/> status is correct', () => {
-  const wrapper = mount(<Game />);
-  const firstPlayer = wrapper
-    .find('div.gameInfo')
-    .children()
-    .first()
-    .text();
-  expect(firstPlayer).toEqual('Next player is X');
+describe('Game', () => {
```
Comment (strange-developer) [inline src/components/game/game.test.jsx]: The describe naming convention

File: src/components/game/game.test.jsx [hunk 6]
```diff
@@ -1,46 +1,52 @@
...
-    .first()
-    .simulate('click');
-  const secondPlayer = wrapper
-    .find('div.gameInfo')
-    .children()
-    .first()
-    .text();
-  expect(secondPlayer).toEqual('Next player is O');
+  it('status is correct', () => {
+    const comp = mount(<Game />);
```
Comment (strange-developer) [inline src/components/game/game.test.jsx]: Why the preference of `mount` as opposed to `shallow` here?

**Review by strange-developer — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (hyperxtend) [reply package.json]: No, it should be under dependencies. I made the change.

File: package.json [hunk 7]
```diff
@@ -8,7 +8,8 @@
     "react-bootstrap": "^1.0.0-beta.17",
     "react-dom": "^16.13.1",
     "react-scripts": "3.4.0",
-    "react-test-renderer": "^16.13.1"
+    "react-test-renderer": "^16.13.1",
+    "@testing-library/jest-dom": "^5.5.0"
```
Comment (strange-developer) [inline package.json]: Are these dependencies required to run the app?

File: package.json (hunk 1 above)
Comment (strange-developer) [reply package.json]: Is the `bootstrap` dependency required when running the app?

File: src/components/game/game.test.jsx (hunk 6 above)
Comment (hyperxtend) [reply src/components/game/game.test.jsx]: I was referring to the component itself but I see now that using shallow would be preferred as I am testing something that will be rendered.

File: src/components/board/board.test.jsx (hunk 3 above)
Comment (hyperxtend) [reply src/components/board/board.test.jsx]: Added assertion to check if component exists

File: src/components/square/square.test.jsx [hunk 8]
```diff
@@ -1,7 +1,10 @@
...
-import Square from './square';
 import { shallow } from 'enzyme';
+import Square from './square';
 
-it('<Square/> renders without error', () => {
-  shallow(<Square />);
+describe('<Square/>', () => {
+it('checks if component exists', () => {
+  const component = shallow(<Square />);
+  expect(component.exists());
```
Comment (strange-developer) [inline src/components/square/square.test.jsx]: `.toBe(true)`

File: src/components/game/game.test.jsx [hunk 9]
```diff
@@ -1,46 +1,53 @@
...
-  const firstPlayer = wrapper
-    .find('div.gameInfo')
-    .children()
-    .first()
-    .text();
-  expect(firstPlayer).toEqual('Next player is X');
+describe('<Game/>', () => {
+  it('checks if component exists', () => {
+    const component = shallow(<Game />);
+    expect(component.exists());
```
Comment (strange-developer) [inline src/components/game/game.test.jsx]: `.toBe(true);`

File: src/components/board/board.test.jsx [hunk 10]
```diff
@@ -1,19 +1,22 @@
...
 
-it('<Board/> renders without error', () => {
-  let squares = Array(9).fill(null);
-  shallow(<Board squares={squares} />);
-});
+describe('<Board/>', () => {
+  it('checks if component exists', () => {
+    const squares = Array(9).fill(null);
+    const component = shallow(<Board squares={squares} />);
+    expect(component.exists());
```
Comment (strange-developer) [inline src/components/board/board.test.jsx]: .toBe(true)

**Review by strange-developer — CHANGES_REQUESTED** [review]

File: src/components/square/square.test.jsx (hunk 8 above)
Comment (hyperxtend) [reply src/components/square/square.test.jsx]: Will add it to tests

File: package.json (hunk 1 above)
Comment (hyperxtend) [reply package.json]: Only for the app's layout if it is on mobile, but I can take it out completely and just use react-bootstrap

File: package.json [hunk 11]
```diff
@@ -39,14 +34,12 @@
...
     "enzyme-to-json": "^3.4.4",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-plugin-import": "^2.20.2",
     "gh-pages": "^2.2.0",
-    "@testing-library/jest-dom": "^4.2.4",
-    "@testing-library/react": "^9.5.0",
-    "@testing-library/user-event": "^7.2.1",
-    "bootstrap": "^4.4.1",
     "qa-utilities": "^1.1.0"
```
Comment (strange-developer) [inline package.json]: Is this required here?

**Review by strange-developer — CHANGES_REQUESTED** [review]

**Review by strange-developer — APPROVED** [review]

**Pull request merged**